.storybook/stories/tree/tree.stories.ts

```diff
@@ -26,9 +26,10 @@ const defaultParameters: Parameters = {
   argTypes: {
     // inputs
     clrLazy: { control: { disable: true } },
+    asLink: { defaultValue: false, control: { type: 'boolean' } },
```

Comment:
```suggestion
    // story helpers
    asLink: { defaultValue: false, control: { type: 'boolean' } },
```

Separate the story helpers from the component inputs.

Reply:
corrected in commit e22dd2e